docs/pages/versions/unversioned/index.md

```diff
@@ -33,7 +33,7 @@ The easiest way to create a bare React Native app with support for the Expo SDK
 Every quarter there is a new Expo SDK release that typically updates to the latest stable version of React Native and includes a variety of bugfixes, features and improvements to the Expo SDK.
 
 | Expo SDK Version | React Native Version |
-| ---------------- | :------------------: |
+| ---------------- | -------------------- |
```

Comment:
`:-:` is the GFM way to center the column content, it doesn't affect docs website display, but works when viewing file on GitHub. 

The redesigned Table component, which is a part of the upcoming docs redesign will support columns align, so I think it won't hurt to just leave this in for now.

Reply:
Well, the behavior is that it centers that column header only, but not the contents, nor the other column header, so it's just inconsistent...

Reply:
![image](https://user-images.githubusercontent.com/852069/160245589-cb80b5e9-87cd-4d12-99dc-e8085ef7ddcc.png)

Reply:
I see, let's change this for now. 👍